### ISSUE TITLE
On visiting IP of ATC only get heading text

### ISSUE THREAD

**Issue by biosed** [facebook/augmented-traffic-control#102]
When I visit http://ip_of_atc:8080/atc_demo_ui/ I only get "Augmented Traffic Control Demo UI"

Centos 6.5 (64bit)
Python 2.7.9
iproute 2.6.32

Installed via pip

**Comment by chantra** [comment]
Hi @biosed 
Have you installed all the modules? like `django-atc-api` ... ?

Did you had them to INSTALLED_APPS and urls.py?

Did you run django's migration script? `python manage.py migrate`

What happen when you go to  http://ip_of_atc:8080/api/v1/shape ? Do you get a page from django rest framework? or any error?

**Comment by biosed** [comment]
Hi @chantra 

installed all the modules listed in the docs

Have added this to atcui/settings.py
` # Django ATC API
    'rest_framework',
    'atc_api',
    # Django ATC Demo UI
    'bootstrap_themes',
    'django_static_jquery',
    'atc_demo_ui',
    # Django ATC Profile Storage
    'atc_profile_storage',
`

Done the migrate

At http://ip_of_atc:8080/api/v1/shape I get the framework and I am able to shape traffic with json.

No errors

**Comment by chantra** [comment]
So it seems the static files are not being distributed.

Can you check why. Using chrome developer tool should help finding what goes on. Basically we need to find why the files from:
```
    <script src="/static/vendor/react/react-0.12.2.min.js"></script>
    <script src="/static/vendor/react/JSXTransformer-0.12.2.js"></script>
    <script type="text/jsx" src="/static/js/atc-api.js"></script>
    <script type="text/jsx" src="/static/js/atc-profile-storage.js"></script>
    <script type="text/jsx" src="/static/js/atc.js"></script>
    <link rel="stylesheet" href="/static/css/atc.css" />
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="/static/static_jquery/js/jquery.min.js"></script>
```

are not being distributed.

If you run django in debug mode and have `django.contrib.staticfiles` in your installed apps, this is the default.

Try to access those .js files and confirm that you are not getting a 404.

**Comment by biosed** [comment]
I opened it with safari and used remote debugging (I currently cannot access the vlan it is on).

I get these errors:

![image](https://cloud.githubusercontent.com/assets/5225187/7377396/fff01c00-eddd-11e4-83ca-abf4c5192d52.png)

**Comment by chantra** [comment]
@biosed 
Can you check that **ALL** the .js file are loading. It looks like `/static/js/atc.js` is not loading in your specific case but please test each of the .js files frpm the list above.

**Comment by biosed** [comment]
@chantra 

Seems to be loading that file, still have error.

![image](https://cloud.githubusercontent.com/assets/5225187/7377808/b58734e8-ede0-11e4-82e7-60028a29e5c1.png)

**Comment by chantra** [comment]
Hum, I really cant repro this even after a fresh deploy and using safari.
Let's rule out the browser... can you reproduce this on chrome and firefox?
Can you repro after flushing the cache?

**Comment by chantra** [comment]
Hi @biosed 

Did you get a chance to try different web browsers? Does that fix your issue?

**Comment by handv** [comment]
I had the same question. Finally i found chrome is the best browsers to show the entire message.

**Comment by biosed** [comment]
Hey @chantra 

So if I visit http://ip_of_atc:8080/atc_demo_ui/ in desktop chrome it works. Doesn't work in safari or chrome on ipad. Is there a UI for mobile devices to enable shaping?

**Comment by chantra** [comment]
@biosed so I don't have an ios device to test on handy here.

Is this only happening only on 1 ipad? or any ipad? The fact that it affects any browser on ipad is puzzling. I would assume the javascript engine of chrome on ipad is the same than the one on desktop.
I know this works correctly on Android. If you have a chance to use another ios device and confirm this affect all of them, please let me know.

**Comment by chantra** [comment]
@biosed so I have tried Safari on an iPad and could not repro so I can help much.
You may want to use the javascript console to find out what happens.

**Comment by ankit-ankuagrawal** [comment]
some problem facing when accessing on android device using crome browser for the setup

**Comment by zfjagann** [comment]
I tried both Safari and Chrome on an iPad, and Chrome on a Nexus 5, they all worked.

I also tried it using mobile simulation on my desktop chrome, which also works.

Could you be more specific about which version of chrome you're using, or check the developer console to see if there are any errors, or any requests not completing successfully? Otherwise I'm not sure there's much we can do to help here.

**Comment by biosed** [comment]
It seems to be older devices even with updated browsers. For these we are using the API so not a major issue. I can provide any info you need but if you cannot repo perhaps this could be closed and put down to older versions of OS / Browser.

**Comment by zfjagann** [comment]
@biosed: I agree. There's not much we can do to help if we can't reproduce it.

I'm going to close this since it's not blocking you. Let me know if you encounter further issues. Thanks!

**Comment by chantra** [comment]
Could you provide the device type + iOS version.in case we manage to access one of those device we could try to investigate further.thanks

**Comment by chantra** [comment]
pinging this issue as this should be fixed as part of #126
